test/unit/vl-map-draw-action.test.html

```diff
@@ -34,16 +34,17 @@
         await customElements.whenDefined('vl-map-draw-polygon-action');
       });
 
-      test('nadat de tekenactie voltooid is, zal de callback worden opgeroepen', async () => {
+      test('nadat de tekenactie voltooid is, zal de callback worden opgeroepen met de feature als eerste argument en een reject callback als tweede argument', async () => {
```

Comment:
moet ge niet testen dat ook effectief de feature niet wordt toegevoegd als ge de reject callback aanroept?

Reply:
Da's hopelijk iets wat getest is in vl-mapactions of wherever die rejectCallback vandaan komt?